packages/wrangler/src/pages/dev.tsx

```diff
@@ -292,6 +299,7 @@ export const Handler = async ({
     kvPersist: true,
     durableObjectsPersist: true,
     cachePersist: true,
+    envPath: devVarsPath,
```

Comment:
I think we should use [straight bindings](https://miniflare.dev/core/variables-secrets/#bindings) like wrangler proper does: https://github.com/cloudflare/wrangler2/blob/57cf14e718749ab43f70034e21e600692cc90727/packages/wrangler/src/index.tsx#L1164

And ideally the `getVarsForDev` function that they've created as well.

This'll mean that Miniflare doesn't need to know anything about the fs, and will possibly make running wrangler in some other workdir easier in the future.